_sass/custom/custom.scss

```diff
@@ -1253,12 +1622,12 @@ main {
 
   @include mq(md) {
     display: block;
-    padding: 1px 0;
-    position: absolute;
-    left: 0;
-    right: 0;
-    z-index: 2;
-    top: 0;
+    // padding: 1px 0;
+    // position: absolute;
+    // left: 0;
+    // right: 0;
+    // z-index: 2;
+    // top: 0;
```

Comment:
In my experience, commenting things out is usually unnecessary. If we need to roll things back, we can just look in Git history.

Reply:
Oops! My appologies. The intention was to have deleted anything commented out during development prior to commit. This code and another commented out block have been removed.